docs/tutorial.rst

```diff
@@ -1212,34 +1212,19 @@ We must also decide the time the mutation will be added, when selection will
 start and at what frequency we want our selected mutation to be at the end of
 the simulation.
 
-Let's assume the mutation appeared 1000 generations ago, it has a positive
-effect on fitness (s=0.5). Also, we want the mutation to have reached a frequency
-of at least 0.8 by the end. Next, we'll walk through the steps required to do this:
-
-.. note::
-
-    Note that because we are doing a forward-in-time simulation, you should be
-    careful with your conditioning. For example, even a strongly selected mutation
-    would not be able to reach 80% frequency in just a few generations. Since
-    this conditioning works by re-running the simulation until the condition is
-    achieved, a nearly impossible condition will result in very long run times.
-
-To set things up, we need to first add the site at which the selected mutation
-will occur.  This is like adding a DFE, except to a single site -- we're saying
-that there is a potential mutation at a particular site with defined fitness
-consequences. So that we can refer to the single site later, we give it a
-unique string ID. Here, we'll add the site in the middle of the contig, with ID
-"hard sweep".
+First, we need to add the site at which the selected mutation will occur.  This
+is like adding a DFE, except to a single site -- we're saying that there is a
+potential mutation at a particular site with defined fitness consequences. So
+that we can refer to the single site later, we give it a unique string ID.
+Here, we'll add the site in the middle of the contig, with ID "hard sweep".
```

Comment:
```suggestion
Here, we'll add the site in the middle of the contig, with ID "hard sweep",
so named because we will imagine this beneficial mutation originates at 
frequency 1 / 2N.
```

maybe we could spell this out for users?